docs/error-codes.md

```diff
@@ -73,6 +73,10 @@ the error code. For example:
 
 - Invalid value 'value' was used for command-line option '--generate-warning-suppressions'; must be 'cs' or 'xml'.
 
+#### `IL1018`: "Invalid value 'value' was used as warning code
+
+- All warning codes must start with `IL` prefix followed by numeric code.
```

Comment:
Should we mention the valid code range for warnings?

Reply:
Not sure it's worth this, the whole file is about error/warning codes